### ISSUE TITLE
Add filtering on the rooms

### ISSUE THREAD

**Issue by ColonelPhantom** [svthalia/sagexit#25]
I think it would be useful to be able to filter by room in Mercator, so that it is easier to see where there is still a spot free. Color-coding may also help this, but I think filtering is more helpful.

**Comment by JobDoesburg** [comment]
Agreed that would be great. Ideally we would use fullcalendar.io 'resources' for this, however this is only available in the crazy expensive premium bundle. 

So we would need to build this ourselves

**Comment by ColonelPhantom** [comment]
I have looked at the pricing for fullcalendar, and it seems like it is free for non-profits (is Thalia registered as a nonprofit?), and for GPLv3 projects. Given that this repo is under the AGPLv3, I am not sure if using that key would be legal?

**Comment by JobDoesburg** [comment]
Oeh did not know that. I think that would definitely be a solution! (yes we're non-profit but we're also AGPLv3)
